### PR TITLE
Issue resolved #128 #129 #130 #131

### LINKED ISSUE #128: 13. Alignment of Sign In Button to Menu Button of Navbar on EES main page

### PR DESCRIPTION
**Pull request by mdathar4403**
#128  #129  #130  #131  issue resolved

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { GoogleOAuthProvider } from '@react-oauth/google';
 // import Udyam from './Udyam/Udyam';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
-const clientId = process.env.REACT_APP_CLIENT_ID;
+const clientId = 'test';
```
Comment (Ankur-Agrawal-ece20) [inline src/index.js:9]: Change this back to the previous line.

**Review by Ankur-Agrawal-ece20 — CHANGES_REQUESTED** [review]

**Comment by mdathar4403** [comment]
ees Navbar corrected

File: src/index.js (hunk 1 above)
Comment (Ankur-Agrawal-ece20) [reply src/index.js:10]: const clientId = process.env.REACT_APP_CLIENT_ID;
this should be the code

**Review by Ankur-Agrawal-ece20 — CHANGES_REQUESTED** [review]

**Review by RainaJain5 — CHANGES_REQUESTED** [review]
Add a light color background behind navbar and increase space between all elements of navbar. 
Rename images in mashal events as per the file name added.

**Review by Ankur-Agrawal-ece20 — APPROVED** [review]

**Pull request merged**